tests/lib/__init__.py

```diff
@@ -447,6 +446,7 @@ def _check_stderr(
 
     lines = stderr.splitlines()
     for line in lines:
+        line = line.strip()
```

Comment:
Probably not relevant but should this be `lstrip` instead?

Reply:
done